article/app/services/dotcomponents/ArticlePicker.scala

```diff
@@ -176,14 +174,16 @@ object ArticlePicker {
       LocalRenderArticle
     } else if (dcrShouldRender(request)) {
       RemoteRender
-    } else if (dcrCouldRender(page, request) && userIsInCohort) {
+    } else if (dcrCouldRender(page, request) && userIsInDotcomRenderingCohort && ArticlePageChecks.isDiscussionDisabled(page)) {
+      RemoteRender
+    } else if (dcrCouldRender(page, request) && userIsInDiscussionRenderingCohort) {
       RemoteRender
     } else {
       LocalRenderArticle
     }
 
     // include features that we wish to log but not whitelist against
-    val features = whitelistFeatures + ("userIsInCohort" -> userIsInCohort) + ("isAdFree" -> isAddFree) + ("isArticle100PercentPage" -> isArticle100PercentPage)
+    val features = whitelistFeatures + ("userIsInCohort" -> userIsInDotcomRenderingCohort) + ("isAdFree" -> isAddFree) + ("isArticle100PercentPage" -> isArticle100PercentPage)
```

Comment:
Would you be able to add the `discussionRenderingCohort` key value here so we can log it?

Reply:
Certainly